### PR TITLE
Updated header + sidenav/accordion 

### PR DESCRIPTION
**Pull request by FSHA**
@SvenDowideit @jsouth @sanscontext

### CODE REVIEW

**Comment by SvenDowideit** [comment]
firefox_screenshot_2016-07-12t23-50-46 324z

looks like something has made the heading anchors be shown all the time (the onhover reveal is busted)

(firefox on linux)

**Comment by SvenDowideit** [comment]
I'm finding it harder to see what menu elements are "menu" and which actually go to a page.

the light grey is also hard for me to read, presumably because its so much lighter than the rest of the page?

**Comment by SvenDowideit** [comment]
LGTM once the para marker hover is fixed.

File: themes/docker-2016/static/documentation.css [hunk 1]
```diff
@@ -70,18 +75,6 @@
     margin: 1.25rem 0;
     width: 100%;
 }
-
-a.anchorLink {
-    margin-left: 5px;
-    visibility: hidden;
```
Comment (SvenDowideit) [inline themes/docker-2016/static/documentation.css]: this is (i think) what's needed - or at least make the markers less BOLD and BRIGHT

**Comment by SvenDowideit** [comment]
mmm, actually, I think you'll do the menu differently when you see what it's like in full :/

I need way more visual hints to navigate a menu this massive :(

**Comment by sanscontext** [comment]
@SvenDowideit, any suggestions on what would convey what's currently missing?

**Comment by SvenDowideit** [comment]
for this PR - no, the only thing that needs fixing pre-merge is the para anchors

for next iteration, I think I need to talk to @FSHA so we can work out what my problem is without me pre-supposing a solution.

File: themes/docker-2016/static/documentation.css (hunk 1 above)
Comment (FSHA) [inline themes/docker-2016/static/documentation.css]: added the CSS back into documentation.css

Thanks for the heads up!

**Comment by SvenDowideit** [comment]
LGTM

**Comment by sanscontext** [comment]
LGTM

**Pull request merged**